package.json

```diff
@@ -39,6 +39,16 @@
                     ".bal"
                 ],
                 "configuration": "./grammar/ballerina.configuration.json"
+            },
+            {
+                "id": "toml",
```

Comment:
Do we need to language contributor, configurationDefaults, and the toml scheme if we use the currently open bal project to get its toml file?